src/olympia/files/models.py

```diff
@@ -706,6 +724,12 @@ class Meta:
         db_table = 'validation_annotations'
 
 
+class WebextPermission(ModelBase):
+    permissions = JSONField(default={})
+    file = models.ForeignKey('File', related_name='_webext_permissions',
```

Comment:
I'd make this a `OneToOneField` and adjust the code above to match. We should never have a `File` with multiple `WebextPermission`s